docs/userguide/torch_forecasting_models.md

```diff
@@ -313,53 +313,102 @@ to see examples of how to do it.
+| `NBEATSModel` | Energy  | 32    | yes  | 32         | 0           | 41.5s          |
+| `NBEATSModel` | Energy  | 32    | yes  | 32         | 2           | 40.6s          |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 0           | 2.8s           |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 2           | 1.65           |
+| `NBEATSModel` | Energy  | 32    | yes  | 1024       | 4           | 1.8s           |
+|               |         |       |      |            |             |                |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 0           | 78s            |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 2           | 72s            |
+| `TFTModel`    | Energy  | 64    | no   | 32         | 4           | 72s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 0           | 46s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 2           | 38s            |
+| `TFTModel`    | Energy  | 64    | no   | 1024       | 4           | 39s            |
+| `TFTModel`    | Energy  | 64    | yes  | 32         | 0           | 125s           |
+| `TFTModel`    | Energy  | 64    | yes  | 32         | 2           | 115s           |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 0           | 59s            |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 2           | 50s            |
+| `TFTModel`    | Energy  | 64    | yes  | 1024       | 4           | 50s            |
+|               |         |       |      |            |             |                |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 0           | 70s            |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 2           | 62.6s          |
+| `TFTModel`    | Energy  | 32    | no   | 32         | 4           | 63.6           |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 0           | 31.9s          |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 2           | 45s            |
+| `TFTModel`    | Energy  | 32    | no   | 1024       | 4           | 44s            |
+| `TFTModel`    | Energy  | 32    | yes  | 32         | 0           | 73s            |
+| `TFTModel`    | Energy  | 32    | yes  | 32         | 2           | 58s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 0           | 41s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 2           | 31s            |
+| `TFTModel`    | Energy  | 32    | yes  | 1024       | 4           | 31s            |
+
+-------------
+
+## Early Stop
+Early stop can be an efficient way to cut down on the training time.  Using early stop trains the model for fewer epochs by stopping the training when the validation loss does not improve for a set number of `nr_epochs_val_period`.
+
+In this example the model reaches the lowest val_loss at epoch 10 and finishes training at epoch 15. 
```

Comment:
Are you sure this will always be the case even with random initialisations? Perhaps you don't need to mention this, but instead you could write something explaining where to pay attention to in the code below - something like

```
Early stopping can be easily implemented on all Darts neural networks based models, by leveraging PyTorch Lightning's `EarlyStopping` callback, as shown in the example code below.
```